### ISSUE TITLE
Tagline

### ISSUE THREAD

**Issue by lawvu-dan** [jezzsantos/saastack#13]
_Building the hard things so you don't have to_

**Comment by jezzsantos** [comment]
Yeah, that's cool, but it doesn't really say much more about what this thing is.
Does that matter?

**Comment by lawvu-dan** [comment]
Ok that's fair. How about "_SaaSCanvas - the SaaS start up Open Source boilerplate solution. We've built the hard stuff so you don't have to"_
